test/test_markdown_parser.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+
+from src.markdown_parser import convert_github_markdown_to_asana_xml
+
+
+class TestConvertGithubMarkdownToAsanaXml(unittest.TestCase):
+    def test_basic_markdown(self):
+        md = """~~strike~~ **bold** _italic_ `code` [link](asana.com)"""
```

Comment:
Have you tested a couple of the examples in https://app.asana.com/0/780306770840675/1200376931872355/f that broke the old version?

Reply:
Good call, I'll do that before merging

Reply:
Tested this on most of the comments and added one change to handle the fact that the Asana API can't handle <pre> tags